src/app.ts

```diff
@@ -54,6 +54,20 @@ export interface AppProps {
   readonly recordConstructMetadata?: boolean;
 }
 
+class SynthRequestCache {
+  public NodeChildrenCache: Map<Node, IConstruct[]> = new Map<Node, IConstruct[]>();
+
+  public NodeAllChildren(node: Node): IConstruct[] {
```

Comment:
To make it more consistent with the original constructs API. 

```suggestion
  public findAll(node: Node): IConstruct[] {
```